components/script/dom/webidls/GPUBindGroupLayout.webidl

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// https://gpuweb.github.io/gpuweb/#gpubindgrouplayout
+[Exposed=(Window, DedicatedWorker), Serializable, Pref="dom.webgpu.enabled"]
+interface GPUBindGroupLayout {
+};
+GPUBindGroupLayout includes GPUObjectBase;
+
+dictionary GPUBindGroupLayoutDescriptor : GPUObjectDescriptorBase {
+    required sequence<GPUBindGroupLayoutBindings> bindings;
+};
+
+// Note: Servo codegen doesn't like the name `GPUBindGroupLayoutBinding` because it's already occupied
```

Comment:
What's the full error message? What's the source of the duplicate name?

Reply:
When the codegen generates the internal `Binding` from the WebIDL for the `GPUBindGroupLayout` interface, the internal name for this bind will be `GPUBindGroupLayoutBinding` and when the codegen tries to generate the `GPUBindGroupLayoutBinding` dictionary we got a duplicate name.
The error message: `GPUBindGroupLayoutBinding` must be defined only once in the type namespace of this module.